src/transformState.js

```diff
@@ -5,7 +5,32 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  const action = Object.values(actions);
+
+  for (const key of action) {
+    switch (key.type) {
+      case 'addProperties':
+        for (const property in key.extraData) {
+          state[property] = key.extraData[property];
+        }
+        break;
+
+      case 'removeProperties':
```

Comment:
Move all cases to the object.

Reply:
I don't understand what you mean. Can you clarify?